v3iofs/fs.py

```diff
@@ -12,17 +12,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
+import logging
+
 from datetime import datetime, timezone
 from http import HTTPStatus
 from os import environ
 from urllib.parse import urlparse
 
 from fsspec.spec import AbstractFileSystem
 from v3io.dataplane import Client
+import sys
```

Comment:
This is unused import. Please run `make test` to check that all sources pass `flake8`